README.md

```diff
@@ -34,6 +34,7 @@ To put the rM2 in recovery, you will need:
 ## Troubleshooting
 * It is **very** important to use a pogo connector because it is almost impossible to achieve a stable connection in another way.
 * Only if the device is in recovery mode there will be messages in the `dmesg` output.
+* If you have red messages like "Cannot enable. Maybe the USB cable is bad?" or your pogo pins contacting makes normal boot with female USB-C breakout board, you could change USB-C cable to 3.2 gen 2 that supports data transfer or use male USB-C breakout board
```

Comment:
3.2 gen 2 would not be a proper solution, as you can use older spec cables as well. It just needs to be a cable that supports data transfer so that it has the required pins. The pins we are working with are the secondary bus, so they would just need to not be a usb 2 cable.

https://pinoutguide.com/Slots/usb-type-c_pinout.shtml